docs/_source/tutorials-examples/population-synthesis/evolve_single_binaries.ipynb

```diff
@@ -0,0 +1,496 @@
+    "print(BINARY.star_2.natal_kick_array)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will be exactly the same evolution as the previous one\n",
+    "BINARY.to_df(extra_columns={'step_names':'string'})[col]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Changing the kick\n",
+    "\n",
+    "We will restore the `BINARY` to its initial state and increase the first kick velocity to disrupt the binary after the first supernova."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.restore()\n",
+    "BINARY.to_df()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# [velocity, azimuthal angle, polar angle, phase]\n",
+    "BINARY.star_1.natal_kick_array = [1000., 0.858129274334538, 1.9157148786534735, 1.8675467897282945]\n",
+    "BINARY.star_2.natal_kick_array = [None, None, None, None] # default\n",
+    "print(BINARY.star_1.natal_kick_array)\n",
+    "print(BINARY.star_2.natal_kick_array)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.evolve()\n",
+    "BINARY.to_df(extra_columns={'step_names':'string'})[col]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Evolve from specific step\n",
+    "\n",
+    "You can provide the row number (starting at 0), to reset the binary to that specific evolutionary phase.\n",
+    "This can be useful for debugging a specific evolutionary state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 69,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BINARY.restore(3)\n",
+    "BINARY.to_df()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading a binary from an existing population"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Assume we run the population somewhere else and you have the h5 file associated with it, and you want to load the population in memory and re-evolve a binary. \n",
+    "We will have to load the `population_params.ini` file associated with the Population file to re-evolve the binary in the same way.\n",
+    "\n",
+    "We will use the `BBH_contact.h5` file in the example dataset and use the standard `population_params.ini` SimulationProperties from earlier in this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from posydon.popsyn.synthetic_population import Population\n",
+    "from posydon.config import PATH_TO_POSYDON_DATA\n",
+    "\n",
+    "data_path = f'{PATH_TO_POSYDON_DATA}/POSYDON_data/tutorial/population-synthesis/examples/'\n",
```

Comment:
Was there a decision that we add tutorial data to the POSYDON_data set? If so, this needs to be told to Zepei how will provide the new v1 set to Tassos.

Reply:
I suggest we do not add test populations as part of the POSYDON dataset and distribute is independently.

Reply:
OK, the above path only suggested that it is part of the POSYDON data. So, may use a different path to make clear where to find it actually.